tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs

```diff
@@ -15,26 +15,35 @@ public class Response
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public List<string> ResponseErrors { get; set; }
 
+    [JsonPropertyName("next_steps")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public List<string>? NextSteps { get; set; }
+
     protected string ToString(StringBuilder value)
     {
         return ToString(value.ToString());
     }
 
     protected string ToString(string value)
     {
-        List<string> errors = [];
+        List<string> messages = [];
         if (!string.IsNullOrEmpty(ResponseError))
         {
-            errors.Add("[ERROR] " + ResponseError);
+            messages.Add("[ERROR] " + ResponseError);
         }
         foreach (var error in ResponseErrors ?? [])
         {
-            errors.Add("[ERROR] " + error);
+            messages.Add("[ERROR] " + error);
+        }
+
+        foreach (var nextStep in NextSteps ?? [])
+        {
+            messages.Add("[NEXT STEPS] " + nextStep);
```

Comment:
Can this be changed to output like: `NEXT STEPS]: <concatenated steps>

Currently it will look like below and that's confusing.
[NEXT STEPS]: <step 1>
[NEXT STEPS]: <step 2>

Reply:
ah I can also make it a string and not a list?